python/taichi/misc/gui.py

```diff
@@ -248,14 +247,18 @@ def get_key_event(self):
 
         if e.key == GUI.WHEEL:
             e.delta = event.delta
+        else:
+            e.delta = (0, 0)
 
         for mod in ['Shift', 'Alt', 'Control']:
             if self.is_pressed(mod):
                 e.modifier.append(mod)
+
         if e.type == GUI.PRESS:
             self.key_pressed.add(e.key)
         else:
             self.key_pressed.discard(e.key)
```

Comment:
IMHO , moving wheel event into press type might lead to a problem . Once this event is saved into key_pressed set , it will never be deleted . Cause wheel event does not have any corresponding 'Release' operation . I think it is better to treat it as a different type from 'Press'.

Reply:
Sorry , I got a wrong understanding with this code just now . It does not matter if we do not use ```gui.is_pressed(GUI.WHEEL)``` ... Just feel free to handle this _\(:3」∠)\_ @archibate